### ISSUE TITLE
How to migrate with this system?

### ISSUE THREAD

**Issue by chx** [VR-25/tarb#2]
I am trying to migrate between a rooted Pixel 2 and Pixel 4 with Magisk installed. 

1. Plug in old phone
2. `adb push tarb-am64 /sdcard`
3. `adb shell`
4.  `su` (grant access)
1. `sh /sdcard/tarb-arm64 -b .`
3. `adb pull /sdcard/.vr25/tarb`
5. -move cable to the new phone-
8. `adb push tarv /sdcard/.vr25`
9. `sh /sdcard/tarb-arm64 -r`
6. `adb shell`
7. `su` (grant access)

And it restored the applications but not their data, accounts etc.

**Comment by VR-25** [comment]
This is weird.
Did you see `data` and `data_de` in the terminal output while restoring?
Verbose is saved to `/dev/.vr25/tarb.log` (volatile).

**Comment by VR-25** [comment]
One more question, have you verified that each backup actually contains `data.tar.zst`, and that those archives can be extracted  (using an external tool) without errors?

**Comment by chx** [comment]
I verified both. But now I redid it with `tarb -badsm10 . ` as found on https://forum.xda-developers.com/t/tarb-a-backup-solution-for-android-with-recovery-mode-support.4443801/ and it worked (aside from weirdos like Duo Mobile but that thing is deliberately migrate unfriendly). It's weird indeed because I saw `data` and `data_de` on the first try. 

Maybe add that option to the main readme?

Thanks for the amazing utility, I tried with Titanium and totally didn't work. I am now a happy Pixel 4 user.
